src/features/customerList/customerListTableFilters/CustomerListTableFiltersList.tsx

```diff
@@ -0,0 +1,267 @@
+
+  const boatTypesQuery = useQuery<FILTER_BOAT_TYPE_LABELS>(FILTER_BOAT_TYPE_LABELS_QUERY, {
+    skip: arrayFilterIsEmpty(filters.boatTypeIds),
+  });
+  const pierQuery = useQuery<FILTER_PIER_LABEL, FILTER_PIER_LABEL_VAR>(FILTER_PIER_LABEL_QUERY, {
+    skip: !filters.pierId,
+    variables: {
+      pierId: filters.pierId ?? '',
+    },
+  });
+  const berthQuery = useQuery<FILTER_BERTH_LABEL, FILTER_BERTH_LABEL_VAR>(FILTER_BERTH_LABEL_QUERY, {
+    skip: !filters.berthId,
+    variables: {
+      berthId: filters.berthId ?? '',
+    },
+  });
+  const [getHarbor] = useHarbors();
+  const [getWinterStorageArea] = useWinterStorageAreas();
+  const winterStoragePlaceQuery = useQuery<FILTER_WINTER_STORAGE_PLACE_LABEL, FILTER_WINTER_STORAGE_PLACE_LABEL_VAR>(
+    FILTER_WINTER_STORAGE_PLACE_LABEL_QUERY,
+    {
+      skip: !filters.winterStoragePlaceId,
+      variables: {
+        winterStoragePlaceId: filters.winterStoragePlaceId ?? '',
+      },
+    }
+  );
+
+  const resolveWinterStoageAreaId = (_value: FilterEntryValue) => {
+    const value = _value as string;
+    const result = getWinterStorageArea({ winterStorageAreaId: value });
+
+    return result?.data?.winterStorageArea?.properties?.name;
+  };
+  const renderValueMap: Record<
+    keyof CustomerListTableFilters,
+    (value: FilterEntryValue) => string | undefined | null
+  > = {
+    customerGroups: (value: FilterEntryValue) => t(`common.customerGroups.${value}`),
+    boatTypeIds: (value: FilterEntryValue) =>
+      boatTypesQuery.data?.boatTypes?.find((boatType) => boatType.id === (value as string))?.name,
+    leaseStatuses: (value: FilterEntryValue) => t(`common.leaseStatus.${value.toString().toLocaleLowerCase()}`),
+    harborIds: (_value: FilterEntryValue) => {
+      const value = _value as string;
+      const harborResult = getHarbor({ harborId: value });
+
+      return harborResult?.data?.harbor?.properties?.name;
+    },
+    pierId: (_: FilterEntryValue) => pierQuery.data?.pier?.properties?.identifier,
+    berthId: (_: FilterEntryValue) => berthQuery.data?.berth?.number,
+    winterStorageGridAreaIds: resolveWinterStoageAreaId,
+    winterStoragePlaceId: (_: FilterEntryValue) => winterStoragePlaceQuery.data?.winterStoragePlace?.number?.toString(),
+    winterStorageAreaIds: resolveWinterStoageAreaId,
+    dateInterval: (_value: FilterEntryValue) => {
+      const value = _value as string;
+
+      try {
+        const { start, end } = createInterval(value);
+        const startDate = start ? parse(start, 'd.L.yyyy', new Date()) : null;
+        const endDate = end ? parse(end, 'd.L.yyyy', new Date()) : null;
+
+        if (startDate && !endDate) {
+          return `${format(startDate, 'yyyy')}-`;
+        }
+
+        if (!startDate && endDate) {
+          return `-${format(endDate, 'yyyy')}`;
+        }
+
+        if (startDate && endDate) {
+          return `${format(startDate, 'yyyy')} - ${format(endDate, 'yyyy')}`;
```

Comment:
Yes. Screenshot from design:

<img width="736" alt="Screenshot 2022-02-16 at 8 06 59" src="https://user-images.githubusercontent.com/9090689/154206295-a5012e65-bd4b-4868-a36a-222bc3319148.png">

Not 100% that this `yyyy` format wasn't left in by accident, because the format in which the date is given by the user changed from a year into a date at some point.